src/utils/deepLinkParams.ts

```diff
@@ -0,0 +1,24 @@
+export const availableSearchParams = ['instCode', 'tagNo', 'search'];
+
+type LinkParam = { [key: string]: string | null };
+
+export function getDeepLinkParams<T extends LinkParam>(
```

Comment:
availableSearchParams bør type deklareres, slik man ikke kan sende inn random string. Da blir det mye lettere å forstå fra utsiden.
Vi bør i tillegg kun sende inn 1, og slippe å sende inn en hel array hver gang.

Reply:
Men man skal kunne sende inn en "random" string 
availableSearchParams er bare de queryParametrene echo jobber med, moduler må kunne sette og gette sine egne query parametre også
Grunnen til at det er et array er vell så man slipper å kalle metoden 3 ganger hvis man ønsker feks å hente ut query parameter verdiene til search, tagNo OG instCode. 
Men det går jo an å støtte array eller sting, så man slipper å sende inn array med 1 string hvis man kun ønsker 1

Reply:
Skjønner, Foreslår å rename availableSearchParams til defaultSearchParams, syns det er lettere å forstå at det kan være andre ting også da :)

Foreslår og å opprette metoden getDeepLinkParam(string) for å kun hente ut en parameter, så slipper vi merkelig array syntax på utsiden med getLinkParams(['one'])

Reply:
Done ;)